package.json

```diff
@@ -59,7 +59,7 @@
 		"build:browser": "webpack"
 	},
 	"dependencies": {
-		"@apify/consts": "^2.25.0",
+        "@apify/consts": "^2.39.0",
```

Comment:
not sure why we have tabs in here, but we should stay consistent

Reply:
Huh, tabs? :smile: Fixed :shrug: